dw1000/src/hl/sending.rs

```diff
@@ -19,7 +19,7 @@ where
     /// driver, but please note that if you're using the DWM1001 module or
     /// DWM1001-Dev board, that the `dwm1001` crate has explicit support for
     /// this.
-    pub fn wait(&mut self) -> nb::Result<Instant, Error<SPI, CS>> {
+    pub fn wait_transmit(&mut self) -> nb::Result<Instant, Error<SPI, CS>> {
```

Comment:
I changed this from wait to wait_transmit because rust-analyzer doesn't know how to resolve methods on two different impls for the same type.

Reply:
It's probably better to also change the other `wait` method to `wait_receive` then, to keep it consistent.

Reply:
Oh, I see you already did that. Please disregard my comment.